hs_core/models.py

```diff
@@ -1429,14 +1431,19 @@ def supports_folders(self):
         """ returns whether folder operations are supported. Computed for polymorphic types"""
         return False
 
+    # TODO: Why isn't this a regular method? Why does it need to be a class method?
+    # It would seem to me that one only creates a bag after a resource has been created,
+    # so that this would be an instance method....
```

Comment:
@alvacouch Agreed, this sounds to me like an instance method rather than a class method as well.